run_benchmark.py

```
@@ -0,0 +1,151 @@
+#!/usr/bin/env python3
+"""
+Clean benchmark runner - Main entry point for all benchmarks
+Uses only HuggingFace datasets and clean evaluation methods
+"""
+import argparse
+import time
```

Ruff (0.12.2)

`time` imported but unused

Remove unused import: `time`

(F401)